Selenium/src/Life360/DownloadLife360Button.java

```diff
@@ -0,0 +1,45 @@
+package Life360;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.Test;
+import java.util.ArrayList;
+
+/*
+Title:DownloadLife360Button
+1.Navigate to life360 website
+2.click on download button
+3.validate download life360 text
+ */
+public class DownloadLife360Button {
+    private String actualText;
+    private String originalText="Download Life360 for free";
+    @Test
+    public void downloadLife360ForFree() {
+
+        System.setProperty("webdriver.chrome.driver", "C:\\Users\\MohanSai\\Downloads\\chromedriver_win32\\chromedriver.exe");
+        ChromeDriver driver=new ChromeDriver();
+        //Navigate to life 360 website
```

Comment:
change comment line to  //Navigate to life 360 website ->  // Navigate to life 360 website
And apply this change to all comments in the method. Just give space between // & Text